core/src/test/java/org/infinispan/scattered/ScatteredSyncFuncTest.java

```diff
@@ -188,4 +191,19 @@ protected void assertLocalValue(int node, MagicKey key, String expectedValue) {
       assertNotNull(ice);
       assertEquals(expectedValue, ice.getValue());
    }
+
+   @Override
+   public void testMergeFromNonOwner() {
+      // TODO : Add support for ScatteredCaches in functional commands : https://issues.jboss.org/browse/ISPN-8078
+      RuntimeException mergeException = new RuntimeException("hi there");
+
+      try {
```

Comment:
I use it, but in this case I wanted the exception inside the CacheException, so I did it like that.